swaglyrics/__main__.py

```diff
@@ -28,12 +28,19 @@ def main():
 	# 	""")
 	# print('\n')
 
+	program = "Swaglyrics"
 	parser = argparse.ArgumentParser(
+		prog=program,
+		usage='{prog} [options]'.format(prog=program),
 		description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required.")
 
-	parser.add_argument('-t', '--tab', action='store_true', help='Display lyrics in a browser tab.')
-	parser.add_argument('-c', '--cli', action='store_true', help='Display lyrics in the command-line.')
+	# To select either one of the arguments provided in a group
+	group = parser.add_mutually_exclusive_group()
```

Comment:
Not sure why both can't be used together if someone (for some reason) wants to. And it should be `required=True` anyway since at least one of those is necessary.